### PR TITLE
change site

### PR DESCRIPTION
**Pull request by wizDaia**
Corrections have been made for the project's main page, the server part of the application has been made, important variables have been taken out into the config, helpers have been created to create a database

### CODE REVIEW

File: helpers/create_schema.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+CREATE DATABASE IF NOT EXISTS your_database_name;
```
Comment (gsvgit) [inline helpers/create_schema.sql:1]: Strange name of database.

File: config/config.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (gsvgit) [inline config/config.json:1]: Is this config should be empty?

File: server.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    const config = JSON.parse(data);
+    res.json(config);
+  });
+});
+
+app.use(express.static(__dirname));
+
+const PORT = process.env.PORT || 8080;
+app.listen(PORT, () => {
+  console.log(`Сервер запущен на порту ${PORT}`);
```
Comment (gsvgit) [inline server.js]: Please, use English.

**Review by gsvgit — CHANGES_REQUESTED** [review]
1. Please, remove files with local settings.
2. Try to use [this](https://github.com/orgs/community/discussions/54049) instead of submitting Js packages sources to the repo.

File: helpers/create_schema.sql (hunk 1 above)
Comment (wizDaia) [reply helpers/create_schema.sql:1]: This is an auxiliary script that the user can simply run. It was assumed that the name of the database was chosen by the user.

File: config/config.json (hunk 2 above)
Comment (wizDaia) [reply config/config.json:1]: Yes, the config with secrets usually contains secret values, tables/databases/etc names. 
If user need to make any changes to the project, such as the address, table name, or database, they can do so by making changes to the config.

**Comment by wizDaia** [comment]
Not relevant branch.

**Pull request closed without merging**